docs/reference/docs/bulk.asciidoc

```diff
@@ -66,17 +66,97 @@ example of a correct sequence of bulk commands:
 
 [source,js]
 --------------------------------------------------
+POST _bulk
 { "index" : { "_index" : "test", "_type" : "type1", "_id" : "1" } }
 { "field1" : "value1" }
 { "delete" : { "_index" : "test", "_type" : "type1", "_id" : "2" } }
 { "create" : { "_index" : "test", "_type" : "type1", "_id" : "3" } }
 { "field1" : "value3" }
-{ "update" : {"_id" : "1", "_type" : "type1", "_index" : "index1"} }
+{ "update" : {"_id" : "4", "_type" : "type1", "_index" : "index1"} }
 { "doc" : {"field2" : "value2"} }
```

Comment:
Why are you changing this ID?  It's supposed to show that you can update a document that you've just created earlier in the bulk request